lua/hbac/command.lua

```diff
@@ -1,26 +1,38 @@
 local state = require("hbac.state")
+local autocommands = require("hbac.autocommands")
 
 local M = {
-	subcommands = {}
+	subcommands = {},
 }
 
 M.subcommands.close_unpinned = function()
 	local curbufnr = vim.api.nvim_get_current_buf()
 	local buflist = vim.api.nvim_list_bufs()
 	for _, bufnr in ipairs(buflist) do
 		if vim.bo[bufnr].buflisted and bufnr ~= curbufnr and not state.is_pinned(bufnr) then
-			vim.cmd('bd ' .. tostring(bufnr))
+			require("hbac.setup").opts.close_command(bufnr)
 		end
 	end
 	print("Hbac: Closed unpinned buffers")
 end
 
-
 M.subcommands.toggle_pin = function()
 	local bufnr = vim.api.nvim_get_current_buf()
 	print("Hbac: Buffer " .. bufnr .. " " .. state.toggle_pin(bufnr))
 end
 
+M.subcommands.toggle_autoclose = function()
+	if state.autoclose_enabled then
+		state.autoclose_enabled = false
+		autocommands.autoclose.disable()
```

Comment:
something like that?
```suggestion
		pcall(autocommands.autoclose.disable())
		-- if the above command fails, the group probably didn't exist, revert the state to false
        state.autoclose_enabled = false
```

Reply:
Added this to autoclose.disable() instead.